services/core-api/app/api/services/traction_service.py

```diff
@@ -147,22 +148,19 @@ def sign_jsonld_credential_deprecated(
         self,
         did,
         verkey,
-        credential,
-    ):
+        credential: BaseModel,
+    ) -> dict:
         options = {"verificationMethod": did + "#verkey", "proofPurpose": "assertionMethod"}
```

Comment:
We need to be careful with hard coding the `verficationMethod` id, might want to provide the whole verification method instead of the did + hard coded id